playbooks/upgrade_postgresql.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create PG client certificate
+  hosts: web
+  become: true
+  gather_facts: true
+
+  pre_tasks:
+    - name: Stop web service
+      ansible.builtin.service:
+        name: "{{ postgresql_upgrade_related_web_service | default('tomcat') }}"
+        state: stopped
+  roles:
+    - role: mirsg.infrastructure.ssl_certificates
+      vars:
+        ssl_certificate: "{{ postgres_client_ssl_certificate }}" # noqa: var-naming[no-role-prefix]
```

Comment:
is this necessary / will this certificate still be valid after the upgrade?

Reply:
I think the cert will still be valid - the question is: do we need to create new client certs? Not sure.